src/Propulsion.Kafka/Producers.fs

```diff
@@ -14,12 +14,13 @@ type Producer
         /// Default: LZ4
         ?compression,
         // Deprecated; there's a good chance this will be removed
-        ?degreeOfParallelism) =
+        ?degreeOfParallelism,
+        ?config) =
```

Comment:
As its a binary breaking change either way, it probably makes sense to move `customize` down here too and add the `custom` arg, in other words copy https://github.com/jet/FsKafka/blob/master/src/FsKafka/FsKafka.fs#L67-L72 and forward all three args on L23

Reply:
Done, made this change.